ci/publish-oci.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+# Copyright (c) 2025 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
+
+if [ ! -f "${HOME}/.unifi/bin/unifi" ]; then
+  echo "Unifi not found! Exit"
+  exit 1
+fi
+
+if [[ "$#" != 3 ]]; then
+ echo "No enjoy parameters!"
+ echo "Usage: $(basename $0) <staging_dir> <release_tag> <registry>"
+ exit 1
+fi
+
+STAGING_DIR=$1
+RELEASE_TAG=$2
+REGISTRY=$3
+
+info() {
+  (>&2 printf "\e[90m$(basename $0)\e[97m: [\e[0;36mINFO\e[97m]:\e[0m $1")
```

Comment:
This will pose a problem if $1 is a format string as well - you can either change `printf` to `echo` or use `printf "...%s..." "$1"` instead